rest-service/manager_rest/rest/resources_v3_1/manager.py

```diff
@@ -223,12 +232,68 @@ def _get_ca_cert_id(self, sm, _manager):
             return existing_managers_certs.pop()
 
 
-class RabbitMQBrokers(SecuredResource):
+class RabbitMQBrokers(ManagementResource):
     @exceptions_handled
     @marshal_with(models.RabbitMQBroker)
     @paginate
     @authorize('broker_get')
     def get(self, pagination=None):
+        """List brokers from the database."""
         return get_storage_manager().list(
             models.RabbitMQBroker,
         )
+
+    @exceptions_handled
+    @authorize('broker_add')
+    @marshal_with(models.RabbitMQBroker)
+    def post(self):
+        """Add a broker to the database."""
+        _broker = rest_utils.get_json_and_verify_params({
+            'name': {'type': unicode},
+            'address': {'type': unicode},
+            'port': {'type': int, 'optional': True},
+            'networks': {'type': dict, 'optional': True},
+        })
+        sm = get_storage_manager()
+
+        # Get the first broker in the list to get the ca_cert and credentials
+        first_broker = sm.list(models.RabbitMQBroker, None, None).items[0]
```

Comment:
no need for the additional `None`, that's just asking for trouble

Reply:
Ah- filters defaults to None?

Reply:
Yea both do

Reply:
So can just do sm.list(models.RabbitMQBroker)?